core/feed/serializer/opds.py

```diff
@@ -153,6 +153,10 @@ def serialize_work_entry(self, feed_entry: WorkEntryData) -> etree._Element:
                     feed_entry.subtitle.text,
                 )
             )
+        if feed_entry.duration is not None:
+            entry.append(
+                OPDSFeed.E(f"{{{OPDSFeed.ATOM_NS}}}duration", str(feed_entry.duration))
```

Comment:
I think we should align this with the XML OPDS 1.2 feed from DeMarque. Which would mean that we should put it into the DCTERMS_NS, rather then atom. 

Here is how is how it looks when I pull the DeMarque XML feed: 
`<dcterms:duration>57894</dcterms:duration>`